src/label_studio_sdk/converter/exports/brush_to_coco.py

```diff
@@ -0,0 +1,332 @@
+        original_height: Height of the original image
+        
+    Returns:
+        tuple: (segmentation, bbox_list, area_list) in COCO format
+    """
+    # Ensure rle is bytes, not string
+    if isinstance(rle, str):
+        rle = rle.encode('utf-8')
+        
+    # Decode RLE
+    try:
+        rle_binary = brush_module.decode_rle(rle)
+    except Exception as e:
+        logger.warning(f"Failed to decode RLE: {e}")
+        return [], [], []
+    
+    # Reshape to image dimensions with 4 channels (RGBA)
+    try:
+        reshaped_image = np.reshape(rle_binary, [original_height, original_width, 4])
+    except Exception as e:
+        logger.warning(f"Failed to reshape RLE data: {e}")
+        return [], [], []
+    
+    # Use only the alpha channel for contour detection
+    alpha_channel = np.array(reshaped_image[:, :, 3]).astype('uint8')
+    
+    # Find contours
+    contours, _ = cv2.findContours(
+        alpha_channel, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE
+    )
+
+    segmentation = []
+    bbox_list = []
+    area_list = []
+    for contour in contours:
+        # OpenCV contours are in format [[[x1,y1]], [[x2,y2]], ...], 
+        # we need to convert to a flat list [x1,y1,x2,y2,...]
+        contour_points = [point[0] for point in contour]
+        
+        # Flatten to a single list
+        contour_flat = []
+        for point in contour_points:
+            contour_flat.extend(point.tolist())
+        
+        # Check if we have at least 3 points (6 coordinates) for a valid polygon
+        if len(contour_points) >= 3:
+            segmentation.append(contour_flat)
+
+            x, y, w, h = cv2.boundingRect(contour)
+            bbox_list.append([x, y, w, h])   # Format: [top-left x, top-left y, width, height]
+
+            # Compute the area of the contour
+            area = cv2.contourArea(contour)
+            area_list.append(int(area))
+
+    return segmentation, bbox_list, area_list
+
+
+def generate_contour_from_polygon(points, original_width, original_height):
+    """Convert polygon annotations to COCO segmentation format
+    
+    Args:
+        points: List of [x, y] points in percentage format
+        original_width: Width of the original image
+        original_height: Height of the original image
+        
+    Returns:
+        tuple: (segmentation, bbox, area) in COCO format
+    """
+    # Convert from percentage to absolute coordinates
+    modified_points = [
+        [round(x * original_width / 100), round(y * original_height / 100)]
+        for x, y in points
+    ]
+
+    # Initialize lists
+    segmentation = [list(flatten(modified_points))]
+
+    # Convert points to a NumPy array and reshape for OpenCV
+    contour = np.array(modified_points, dtype=np.float32).reshape((-1, 1, 2))
+
+    # Calculate bounding box
+    bbox = cv2.boundingRect(contour)
+    bbox = [int(coordinate) for coordinate in bbox]
+
+    # Compute the area of the contour
+    area = cv2.contourArea(contour)
```

Comment:
Why do we need to use contourArea for polygons, could you please clarify? I thought polygons were contours themselves.